reference/outlook/preview/Recurrence.md

```diff
@@ -0,0 +1,201 @@
+
+##### Requirements
+
+|Requirement|Value|
+|---|---|
+|[Minimum mailbox requirement set version](../tutorial-api-requirement-sets.md)|Preview|
+|[Minimum permission level](https://docs.microsoft.com/outlook/add-ins/understanding-outlook-add-in-permissions)|ReadItem|
+|Applicable Outlook mode|Compose or read|
+
+##### Examples
+
+This example gets the recurrence object of an appointment or meeting request item.
+
+```js
+Office.context.mailbox.item.recurrence.getAsync(callback);
+function callback(asyncResult){
+	var context = asyncResult.context;
+	var recurrence = asyncResult.value;
+	var endDate = recurrence.seriesTime.getEndDate();
+}
+```
+
+The following is an example of the results of the `getAsync` call.
+
+```js
+Recurrence = {"recurrenceType": "weekly","recurrenceProperties": {"interval": 2, "days": ["mon","thu","fri"], "firstDayOfWeek": "sun"}, "seriesTime": {seriesTimeObject}, "recurrenceTimeZone": {"name": "Pacific Standard Time", "offset": -480}}
+```
+
+####  setAsync(recurrencePattern, [options], [callback])
+
+Sets the recurrence pattern of an appointment.
+
+##### Parameters:
+
+|Name|Type|Attributes|Description|
+|---|---|---|---|
+|`recurrencePattern`|[Recurrence](recurrence.md)||A recurrence object.|
+|`options`|Object|&lt;optional&gt;|An object literal that contains one or more of the following properties.|
+|`options.asyncContext`|Object|&lt;optional&gt;|Developers can provide any object they wish to access in the callback method.|
+|`callback`|function|&lt;optional&gt;|When the method completes, the function passed in the `callback` parameter is called with a single parameter, `asyncResult`, which is an [`AsyncResult`](simple-types.md#asyncresult) object.|
+
+##### Errors
+
+|Error code|Description|
+|------------|-------------|
+|`InvalidEndTime`|The appointment end time is before the appointment start time.|
+
+##### Requirements
+
+|Requirement|Value|
+|---|---|
+|[Minimum mailbox requirement set version](../tutorial-api-requirement-sets.md)|Preview|
+|[Minimum permission level](https://docs.microsoft.com/outlook/add-ins/understanding-outlook-add-in-permissions)|ReadWriteItem|
+|Applicable Outlook mode|Compose|
+
+> Note: `setAsync` should only be available for series items and not occurrence items.
+
+##### Examples
+
+The following example sets the recurrence pattern of an appointment series.
+
+```js
+var seriesTimeObject = new seriesTime(); 
+seriesTimeObject.setStartDate(2017,11,2);  
+seriesTimeObject.setEndDate(2017,12,2); 
+seriesTimeObject.setStartTime(10,30); 
+SeriesTimeObject.setDuration(30);
+
+var pattern = {"seriesTime": seriesTimeObject, "type": "Weekly", "properties": {"interval": ", "days": ["Tue", "Thu"], "firstDayOfWeek": "Sun"}, "recurrenceTimeZone": {"name": "Pacific Standard Time"}}; 
```

Comment:
Looks like this closing bracket on line 183 was accidently snipped off.

Reply:
As we discussed, those changes are still included so this is ok.